x-pack/test/apm_api_integration/basic/tests/settings/custom_link.ts

```diff
@@ -139,5 +140,18 @@ export default function customLinksTests({ getService }: FtrProviderContext) {
       expect(status).to.equal(200);
       expect(body).to.eql([]);
     });
+
+    describe('transaction', () => {
+      before(() => esArchiver.load('8.0.0'));
+      after(() => esArchiver.unload('8.0.0'));
+
+      it('fetches a transaction sample', async () => {
+        const response = await supertestRead.get(
+          '/api/apm/settings/custom_links/transaction?service.name=opbeans-java'
+        );
+        expect(response.status).to.be(200);
+        expect(response.body.service.name).to.eql('opbeans-java');
```

Comment:
anything else that we want to/should test?

Reply:
no, only this since I'm passing a query param filtering by this service name.